frappe/public/js/frappe/widgets/utils.js

```diff
@@ -124,7 +124,7 @@ function shorten_number(number, country) {
 	for (const map of number_system) {
 		const condition = map.condition ? map.condition(x) : x >= map.divisor;
 		if (condition) {
-			return Math.round(number/map.divisor) +  ' ' + map.symbol;
+			return (x/map.divisor) +  ' ' + map.symbol;
```

Comment:
@mujeerhashmi This won't work since `number` can be negative in which case this will return the wrong value. Also `number/map.divisor` should be rounded since the number has to be shortened. Instead we can do:
```suggestion
			return (number/map.divisor).toFixed(2) + ' ' + map.symbol;
```